### DISCUSSION TITLE
Add a piece of documentation to i3 user's guide (tabbed / stacking layouts side by side)

### DISCUSSION THREAD

**Discussion by RayZ0rr** [i3/i3#4813] [Ideas]
Carefully reading [i3 user guide](https://i3wm.org/docs/userguide.html) and the following two threads : 
[thread 1](https://faq.i3wm.org/question/4850/why-is-there-no-split-tabbedstacked.1.html)
[thread 2](https://faq.i3wm.org/question/5184/how-can-i-get-two-stacked-containers-side-by-side/index.html)

I found a very useful (for me) keybindings that took some time to figure out (most of the time spent to understand how containers work).
```sh
# Create 'split container' (i3 guide terminology) with tabbed layout
bindsym $mod+o split horizontal, layout tabbed

# Same as above (?)
bindsym $mod+u split vertical, layout tabbed
```
It might be helpful if this is provided in the userguide with mentions/hyperlinks to the section [[command_chaining]](https://i3wm.org/docs/userguide.html#command_chaining) along with it as well as a (warning-type) note that this will behave much differently with anything other than `workspace_layout default`

**Comment by Airblader** [comment]
While this is only a "small" addition that in itself wouldn't be problematic, the problem is that making the user guide a place of opinionated tip sharing also explodes its scope. I would like to limit that scope to teaching the building blocks, though, and only add very specific examples if they teach a concept rather than a use case. The concept of command chaining is explained in its corresponding section already.

That doesn't mean I'm against having a place to share tips. But this is a whole separate maintenance burden that we do not want to get into, and part of the reason why we opened Discussions which e.g. has a "Show & Tell" section where you could create essentially a post with a meaningful title and description, and then answer it yourself for people to find. That seems like a good compromise between having such information available close to i3 (rather than very external) and not adding the overhead of having to maintain it.
